### DISCUSSION TITLE
PR Label에 사용한 언어 추가 건의!

### DISCUSSION THREAD

**Discussion by TonyKim9401** [DaleStudy/leetcode-study#328] [건의사항]
![image](https://github.com/user-attachments/assets/fd3f630f-92ca-40b8-b7c4-362ea086ae1c)
안녕하세요! 한주동안 릿코드 문제 풀고 커밋하며 느낀점 건의하려고 합니다!

아무래도 다들 사용하시는 언어가 다르다 보니 저같은 경우 JAVA로 풀어서 그 외에 문법에 익숙하지가 않아서요.
다른 PR을 보더라도 직접 들어가서 확인하기 전에는 어떤 언어로 풀으셨는지 확인이 힘들더라구요.

따라서, PR시 Label에 사용 언어를 표시할 수 있으면 어떨까 합니다!

(*이미 의견이 반영되어 해결된 건의사항 입니다.)

**Comment by DaleSeo** [comment]
@TonyKim9401 좋은 제안 주셔서 감사합니다!

제가 우선 언어별로 레이블은 생성해두었는데, PR에 레이블를 달아주시는 분도 계시고 안 다시는 분도 계신 것 같아요. 아무래도 모든 멤버 분들이 일관적으로 다 달아주셔야 원하시는 효과를 더 얻을 수 있지 않을까 싶은데 어떻게 생각하세요?

PS. 디스코드에 관련 쓰레드 링크 남겨둡니당.
https://discord.com/channels/775115965964222492/1230098590874210304/1273371380326338591
